### PR TITLE
Move documentation into main repository

### PR DESCRIPTION
**Pull request by graysonwright**
## Problem:

Documentation is being kept at
https://github.com/thoughtbot/administrate-docs.

It is difficult to maintain documentation when it is kept in a separate
repository from the rest of the app.

So far, there has only been one documentation pull request. We'd like to
encourage more people to submit documentation suggestions.

## Solution:

- Move documentation from its original source
 to within the main Administrate repo.
- Copy over all existing styles from the documentation app.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -9,6 +9,9 @@ gem "high_voltage"
 gem "pg"
 gem "unicorn"
 
+gem "markdown-rails"
+gem "redcarpet"
```
Comment (graysonwright) [inline Gemfile]: alph-order

**Comment by graysonwright** [comment]
@danbee the visual regressions are unrelated to this PR. I've written up a description of the problem, along with fixes, in https://github.com/thoughtbot/administrate/pull/408.

We should look into [upgrading our Percy plan](https://percy.io/pricing).

Because the regressions aren't coming from this code, I'm okay moving forward on this.

**Comment by graysonwright** [comment]
Also, because we haven't approved the Percy diffs for this PR, future diffs won't be compared against this. They'll be compared against the last commit on master that was approved. Yay!

File: app/assets/stylesheets/docs.scss [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  padding: 0.4em 0.2em 0;
+
+  &.hljs {
+    $black: #202020;
+    $white: #e0e0dc;
+    $green: #b0bf82;
+    $blue: #8fbdcc;
+    $cyan: #b95c56;
+    $yellow: #f8d29d;
+    $red: #b95c56;
```
Comment (graysonwright) [inline app/assets/stylesheets/docs.scss]: Move these colors to the top of the file, pls. Even better, move them to `variables.scss`

File: app/assets/stylesheets/docs.scss [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  color: $blue;
+  text-decoration: none;
+}
+
+.content {
+  margin-bottom: 2em;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 2em;
+  width: 61em;
```
Comment (graysonwright) [inline app/assets/stylesheets/docs.scss]: that's an odd width...

File: app/views/layouts/docs.html.erb [hunk 4]
```diff
@@ -0,0 +1,38 @@
+<html>
+<head>
+  <%= stylesheet_link_tag "docs", media: "all" %>
+  <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/styles/default.min.css">
+  <link href='//fonts.googleapis.com/css?family=Lato|Source+Code+Pro|Fjalla+One' rel='stylesheet' type='text/css'>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/highlight.min.js"></script>
```
Comment (graysonwright) [inline app/views/layouts/docs.html.erb]: Let's keep the highlight.js import tags next to each other.

**Comment by danbee** [comment]
@graysonwright Apologies for not getting to look at this today.

**Comment by graysonwright** [comment]
oh, no worries! I was working with Tyson a bit on CSS stuff mostly.

**Comment by pmaojo** [comment]
Wouldn't it be good to make the docs responsive?

**Comment by graysonwright** [comment]
@pmaojo - I agree, that would be great!

If you'd like to take a look at that, feel free to open a new PR. This is already closed, so there's not much we can do here.

**Comment by pmaojo** [comment]
Hello @graysonwright I will take a look on that. Where are the documentation files?

**Pull request merged**